MAINTAINERS.md

```diff
@@ -1,73 +1,11 @@
-# Maintainers
-
-Habitat Builder is part of the Habitat ecosystem, built and maintained by
-a community of dedicated humans, listed below.
-
-This file lists how the Builder project is maintained and the humans and
-companies who make up "The Builder Maintainers."
+# Maintenance policy
 
-When making changes to the system, this file tells you who needs to review your
-patch - you need approval of at least one maintainer for the relevant subsystems
-to provide a Github Approval on your pull request. Additionally, you need to not
-receive a veto from a Sub-Lead or the Project Lead.
+The Builder project membership adheres to the [Chef OSS Practices](https://github.com/chef/chef-oss-practices/blob/1a266c3e2d7fddc5bee81071e9e7f59f2b092217/project-membership.md#established-project-members) model.
```

Comment:
Might want to just link to this file on the master branch, rather than a specific commit.

Reply:
Good catch, will update